components/actiongroup/stories/template.js

```diff
@@ -65,102 +65,69 @@ export const OverflowOption = (context) => Container({
 	},
 	content: html`
 		${Container({
-				heading: "Wrap",
-				content: Template({
-					customStyles: { "max-inline-size": "288px" },
-					content: [
-						{
-							iconName: "Edit",
-							iconSet: "workflow",
-							label: "Edit",
-						},
-						{
-							iconName: "Copy",
-							iconSet: "workflow",
-							label: "Copy",
-						},
-						{
-							iconName: "Delete",
-							iconSet: "workflow",
-							label: "Delete",
-						},
-						{
-							iconName: "Cut",
-							iconSet: "workflow",
-							label: "Cut",
-						},
-						{
-							iconName: "Move",
-							iconSet: "workflow",
-							label: "Move",
-						},
-					]
-				})
-			}
-		)}
+			withBorder: false,
```

Comment:
These changes just fixed up some indentation, removed the border from around each container, and I think passed context to the templates, instead of the parent `Container()`